### PR TITLE
TECH-368

### PR DESCRIPTION
**Pull request by trista-d**
- Created uid in Submission table, to match entries in Email table
- Added a check to prevent users from submitting more than one form
- Updated the Name form field and added a message underneath the submit button
- Updated favicon and tab title in form

### CODE REVIEW

**Comment by npham49** [comment]
Did you update the new prisma.schema with the uid?

File: packages/forms-frontend/app/api/form/route.ts [hunk 1]
```diff
@@ -17,6 +17,14 @@ export async function POST(req: NextRequest, res: NextResponse) {
         const json = await req.json()
         console.log(json)
 
+        // check if user submitted a form before
+        const submission = await prisma.submission.findUnique({
+            where: {
+                uid: json.uid
+            }
+        })
+        if (submission) throw new Error("You have already submitted this form")
```
Comment (npham49) [inline packages/forms-frontend/app/api/form/route.ts:26]: Let's also add an error handler on the front-end to show this in the toast error message

Comment (npham49) [reply packages/forms-frontend/app/api/form/route.ts:26]: Might also have to double check if we want them to submit multiple or not, not sure if it's part of the requirements.

**Review by npham49 — CHANGES_REQUESTED** [review]
Have a look at my comments!

File: packages/forms-frontend/app/api/form/route.ts (hunk 1 above)
Comment (trista-d) [reply packages/forms-frontend/app/api/form/route.ts:26]: "You have already submitted this form" shows up in the toast already! Should I ask Priscilla about this functionality on Monday? My reasoning right now is that without it, a single user can bombard a centre with emails. If they can submit multiple forms, we should probably limit it to a certain number.

**Comment by trista-d** [comment]
> Did you update the new prisma.schema with the uid?

I just forgot to commit it, sorry!

File: packages/forms-frontend/app/api/form/route.ts (hunk 1 above)
Comment (npham49) [reply packages/forms-frontend/app/api/form/route.ts:26]: Yes please! I'd also prefer that they submit only once but I can see people making an argument for the case of accidental submissions.

Comment (trista-d) [reply packages/forms-frontend/app/api/form/route.ts:26]: We're good to allow clients to only submit once!

File: packages/forms-frontend/app/layout.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from "react"
 import Providers from "@/utils/provider"
 
 export const metadata = {
-    title: "Create Next App",
+    title: "Contact WorkBC Centre",
     description: "Generated by create next app"
```
Comment (npham49) [inline packages/forms-frontend/app/layout.tsx:7]: Change this to "Short form to contact a WorkBC Centre"

**Review by npham49 — CHANGES_REQUESTED** [review]
Just 1 small comments then you're free to merge your changes

**Pull request merged**